apex/transformer/pipeline_parallel/p2p_communication.py

```diff
@@ -26,11 +26,30 @@
 from apex.transformer.pipeline_parallel._timers import _Timers
 
 
+
+class FutureTensor(object):
```

Comment:
```suggestion
class FutureTensor:
```

`(object)` isn't necessary